job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/WeblogicDeployerContext.groovy

```diff
@@ -0,0 +1,151 @@
+package javaposse.jobdsl.dsl.helpers.publisher
+
+import javaposse.jobdsl.dsl.Context
+import javaposse.jobdsl.dsl.ContextHelper
+import org.apache.commons.lang.RandomStringUtils
+
+/**
+ * <p>DSL Support for the weblogic-deployment-plugin.</p>
+ * <p><a href="https://wiki.jenkins-ci.org/display/JENKINS/WebLogic+Deployer+Plugin">WebLogic Deployer Plugin</a></p>
+ */
+class WeblogicDeployerContext implements Context {
+
+    /**
+     * <p>Enumeration of available deployment stage modes.</p>
+     */
+    static enum WeblogicDeploymentStageModes {
+
+        BY_DEFAULT('bydefault'),
+
+        STAGE('stage'),
+
+        NO_STAGE('nostage'),
+
+        EXTERNAL_STAGE('external_stage')
+
+        private final String stringRepresentation
+
+        WeblogicDeploymentStageModes(String stringRepresentation) {
+            this.stringRepresentation = stringRepresentation
+        }
+
+        @Override
+        String toString() {
+            this.stringRepresentation
+        }
+    }
+
+    List<Node> taskNodes = []
+    List<Node> deploymentPoliciesIdsNodes = []
+
+    boolean mustExitOnFailure = false
+    boolean forceStopOnFirstFailure = false
+    boolean deployingOnlyWhenUpdates = false
+    String deployedProjectsDependencies = ''
+
+    /**
+     * <p>Fails the build if the deployment fails.</p>
+     *
+     * @param mustExitOnFailure true, false (default)
```

Comment:
The API viewer currently does not show the param tags. Mention the default values in the method description like this:

```groovy
/**
 * Fails the build if the deployment fails. Defaults to {@code false}.
 */
```